src/plugins/fs/generic.c

```diff
@@ -701,3 +703,75 @@ gboolean bd_fs_can_repair (const gchar *type, gchar **required_utility, GError *
 gboolean bd_fs_can_set_label (const gchar *type, gchar **required_utility, GError **error) {
     return query_fs_operation (type, BD_FS_LABEL, required_utility, NULL, error);
 }
+
+
+static gboolean fs_freeze (const char *device, gboolean freeze, GError **error) {
+    g_autofree gchar* mountpoint = NULL;
```

Comment:
Are you sure people would always use autoptr-capable compiler?

Reply:
**YES**

Reply:
We've had a little chat about that this morning. Anyway, looking at `glib/gmacros.h` defines, the `g_autofree` symbol is actually undefined when `__GNUC__` is undefined. So this way the compilation would fail, preventing any leaks. So no changes needed.